test/testdata/desugar/keyword_args.rb

```diff
@@ -1,31 +0,0 @@
-# typed: true
```

Comment:
We should almost _never_ delete tests—instead, we should showcase the new behavior of Sorbet on this test.

The only exception is when a test is completely redundant with some other test. In that case, please mention the other test(s) that duplicate the behavior being tested here.

Reply:
This test have been fully moved to `test/testdata/infer/ruby3_keyword_args.rb`:
https://github.com/sorbet/sorbet/blob/d331f62b15b019c12b751e0c49ac2ae599e93999/test/testdata/infer/ruby3_keyword_args.rb#L70-L95

I had to rename `foo` to `bar`, because `foo` was already taken